src/components/static/pages/Faq.tsx

```diff
@@ -28,50 +29,43 @@ export class Faq extends React.Component {
             <FaqWrapper>
               <div className="main-title-faq">Frequently Asked Questions</div>
               <ContentWrapper>
-                <div className="question">What is Nuevo Foundation?</div>
-              </ContentWrapper>
-              <ContentWrapper>
-                <div className="questiontext">
-                  Nuevo Foundation is a nonprofit that is run by a group of
-                  passionate professionals. We empower underrepresented
-                  communities with solutions for future success.
+                <div className="questiontext" id="intro">
```

Comment:
The empty space looks a little strange. I would suggest adding a horizontal line after this paragraph of shade the whole section with a different color to make the distinction that this is describing nuevo. Or add the question header back.

Reply:
@Sunny-Dee we talked to izzy about these comments. We'll address them on a follow up PR. 😄